mappings/net/minecraft/item/Item.mapping

```diff
@@ -191,3 +193,5 @@ CLASS net/minecraft/class_124 net/minecraft/item/Item
 	METHOD method_467 getCraftingReturnItem ()Lnet/minecraft/class_124;
 	METHOD method_468 hasCraftingReturnItem ()Z
 	METHOD method_469 getTranslatedName ()Ljava/lang/String;
+	METHOD method_470 getMetadata (I)I
```

Comment:
Looking at prior versions of Yarn for 1.12 when metadata still existed, it looks like the mapping for it is `getPlacementMetadata`, that seems like it would work for conveying the metadata of the placed block.